Q/Qt/build_tarballs.jl

```diff
@@ -95,8 +98,23 @@ EOT
         export PATH=$(echo "$PATH" | sed -e 's!/opt/bin:!!')
         export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$prefix/share/pkgconfig
         export PKG_CONFIG_LIBDIR=$prefix/lib/pkgconfig
+
+        sed -i 's/^QMAKE_C.*FLAGS.*//' ../qt-everywhere-src-*/qtbase/mkspecs/devices/linux-rasp-pi3-g++/qmake.conf
         
         ../qt-everywhere-src-*/configure QMAKE_LFLAGS=-liconv -platform linux-g++ -device linux-rasp-pi3-g++ -device-option CROSS_COMPILE=/opt/bin/$target- \
+            -extprefix $prefix $commonoptions \
+            -skip qtwinextras -fontconfig -sysroot /opt/$target/bin/../$target/sys-root
+        ;;
+        
+    *aarch64-linux*)
+        apk add g++ linux-headers
+        export PATH=$(echo "$PATH" | sed -e 's!/opt/bin:!!')
+        export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$prefix/share/pkgconfig
```

Comment:
Why? `${prefix}/share/pkgconfig` is already in `${PKG_CONFIG_PATH}`, see https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/16223e035a1b316f3d6e15a2a63733bd36f3935f/src/Runner.jl#L912

Reply:
It gave an error in the past, seems resolved now.